{
  Type: Exception,
  Message:
The prefix has already been used: Existing: {ProjectDirectory}Naming/NamerTests.Conflict.
This is mostly caused by a conflicting combination of `VerifierSettings.DerivePathInfo()`, `UseMethodName.UseDirectory()`, `UseMethodName.UseTypeName()`, and `UseMethodName.UseMethodName()`. Prefix: {ProjectDirectory}Naming/NamerTests.Conflict,
  StackTrace:
at PrefixUnique.CheckPrefixIsUnique(String prefix)
at VerifyTests.FileNameBuilder..ctor(MethodInfo method, Type type, String sourceFile, VerifySettings settings)
at VerifyXunit.Verifier.GetVerifier(VerifySettings settings, String sourceFile)
--- End of stack trace from previous location ---
}